charts/kafka-topic-replicator/chart/values.yaml

```
@@ -0,0 +1,39 @@
+# The kafka bootstrap endpoint to contact for communication with the cluster. Required.
+kafkaBrokerEndpoint: "my-cluster-kafka-bootstrap"
+# The topic where records are read and replicated. Required.
+inputTopic: "topic-test-1"
+# The topic the records are replicated to. Required.
+outputTopic: "test"
+# The delay in miliseconds before the record is replicated. Counted from the creation timestamp of the record.  
+# delayMs: 0 
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/distributed-technologies/helm-charts/kafka-topic-replicator
+  pullPolicy: IfNotPresent
+  tag: "0.1.1"
```

This is not needed and we will likely forgot to update it (ex: https://github.com/distributed-technologies/helm-charts/commit/15280251aa0869e8b4114135fc15ff27905d85a7)
```suggestion
  tag: ""
```

---

Then it would just pull the latest image tag right?
I think we have a general problem in the mono repo when there is an application that needs to be build. 
Maybe we should start using appVersion and version, so that we separate the application and helm chart versions.